posthog/src/main/java/com/posthog/PostHogEventName.kt

```diff
@@ -0,0 +1,24 @@
+package com.posthog
+
+public enum class PostHogEventName(public val nameEvent:String){
+    SNAPSHOT("\$snapshot"),
+    PAGEVIEW("\$pageview"),
+    PAGELEAVE("\$pageleave"),
+    SET("\$set"),
+    IDENTIFY("\$identify"),
+    GROUP_IDENTIFY("\$groupidentify"),
+    CREATE_ALIAS("\$create_alias"),
+    CLIENT_INGESTION_WARNING("\$client_ingestion_warning"),
+    WEB_EXPERIMENT_APPLIED("\$web_experiment_applied"),
+    FEATURE_ENROLLMENT_UPDATE("\$feature_enrollment_update"),
+    FEATURE_FLAG_CALLED("\$feature_flag_called"),
+    SURVEY_DISMISSED("survey dismissed"),
+    SURVEY_SENT("survey sent"),
+    SURVEY_SHOWN("survey shown");
```

Comment:
some of those arent used in this library yet such as PAGEVIEW, PAGELEAVE, etc, lets remove all unnecessary events from this lib for now
you can just search its usage across the repo